src/client/contexts/MealsContext.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from "react";
+
+const MealsContext = React.createContext();
+
+const MealsProvider = ({ children }) => {
+  const [meals, setMeals] = useState([]);
+  const [titles, setTitles] = useState([]);
```

Comment:
Is this state needed? Couldn't we get the same data from `meals`?

Reply:
Initialy I was using it. I will remove it.